packages/open-api-page/index.jsx

```diff
@@ -0,0 +1,91 @@
+import { useState, useRef } from 'react'
+import Link from 'next/link'
+import OperationObject from './partials/operation-object'
+import HashiHead from '@hashicorp/react-head'
+import DocsSidenav from '@hashicorp/react-docs-sidenav'
+import Content from '@hashicorp/react-content'
+import useOnClickOutside from './hooks/use-on-click-outside'
+import s from './style.module.css'
+
+function OpenApiPage({
```

Comment:
Ideally this would be TypeScript, but the original implementation was not, and a bit short on time, so intent is to revisit this again the next time we touch this component.